adafruit_ds18x20.py

```diff
@@ -125,25 +132,25 @@ def _read_temp(self):
             t = -((t ^ 0xFFFF) + 1)
         return t / 16
 
-    def _read_scratch(self):
+    def _read_scratch(self) -> ReadableBuffer:
```

Comment:
Since we're reading into `self._buf` which is a `bytearray`, we can type the return as `bytearray` as well.

Reply:
but what about _write_scratch where the input is also a bytearry and I defined as Writeable buffer?

Reply:
You could use any `WriteableBuffer` with `_write_scratch()`, but the idea of `_read_scratch()` is to read into a `bytearray` and return that.